packages/common-ts/src/common/metrics.ts

```diff
@@ -29,3 +33,29 @@ export class Metrics {
     collectDefaultMetrics(metricsOptions)
   }
 }
+
+export interface MetricsServerOptions {
+  logger: Logger
+  registry: Registry
+  port?: number
+  route?: string
+}
+
+export const createMetricsServer = async (
+  options: MetricsServerOptions
+): Promise<Server> => {
+  const logger = options.logger.child({ component: 'MetricsServer' })
+
+  const app = express()
+
+  app.get(options.route || '/metrics', async (_, res) => {
+    res.status(200).send(await options.registry.metrics())
+  })
+
+  const port = options.port || 7300
+  const server = app.listen(port, () => {
+    logger.info('Listening on port', { port })
```

Comment:
This should specify "metrics server is listening on port" - also specifying the hostname is important to be able to run this inside of a container